services/circleci/circleci.service.js

```diff
@@ -0,0 +1,106 @@
+'use strict'
+
+const Joi = require('joi')
+const BaseJsonService = require('../base-json')
+
+const circleSchema = Joi.array()
+  .items(Joi.object({ status: Joi.string() }))
+  .min(1)
+  .required()
+
+module.exports = class CircleCi extends BaseJsonService {
+  async fetch({ token, type, userRepo, branch }) {
+    let url = `https://circleci.com/api/v1.1/project/${type}/${userRepo}`
+    if (branch != null) {
+      url += `/tree/${branch}`
+    }
+    const query = { filter: 'completed', limit: 1 }
+    if (token) {
+      query['circle-token'] = token
+    }
+    return this._requestJson({
+      url,
+      schema: circleSchema,
+      options: { qs: query },
+      errorMessages: { 404: 'project not found' },
+    })
+  }
+
+  static render({ status, color }) {
+    return { message: status, color: color }
+  }
+
+  transform(data) {
```

Comment:
Is this supposed to be a `static` method? I can't remember what the rule is now. When stuff should/shouldn't be `static` is definitely a topic we need to cover when we write docs.

Reply:
Yea, in general you can make anything static that doesn't use `this`. The exception is if you're overriding superclass method, in which case making it static would break the overriding.